mpqp/core/circuit.py

```diff
@@ -620,6 +622,7 @@ def to_other_language(self, language: Language = Language.QISKIT) -> Union[
 
         Args:
             language: Enum representing the target language.
+            processor_id : Identifier of the processor for cirq.
```

Comment:
I would rename cirq_proc_id, so the user knows directly it is for cirq only, without reading the doc ?